pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -2953,6 +2956,27 @@ type OSSLifecycleRule struct {
 	MarkDeletionAfterDays int32 `json:"markDeletionAfterDays,omitempty" protobuf:"varint,2,opt,name=markDeletionAfterDays"`
 }
 
+// OSSCredentialsConfig specifies the credential configuration for OSS
```

Comment:
Is this required or can the ARN be supplied as labels similar to other providers?